subnet/subnet.go

```diff
@@ -394,6 +396,36 @@ func (sm *SubnetManager) WatchLeases(receiver chan EventBatch, cancel chan bool)
 	}
 }
 
+func (sm *SubnetManager) ListLeases(receiver chan EventBatch, cancel chan bool) {
```

Comment:
Instead of adding functionality to periodically list leases, it's better to keep the list in the backend itself. You can then also keep netlink.Route objects instead of Leases so they're ready to be pushed down into the kernel.